test/Microsoft.Azure.Devices.Edge.Test.Common/EdgeModule.cs

```diff
@@ -74,7 +72,9 @@ async Task WaitForStatusAsync()
                         // Retry if iotedged's management endpoint is still starting up,
                         // and therefore isn't responding to `iotedge list` yet
                         static bool DaemonNotReady(string details) =>
+                            details.Contains("Incorrect function", StringComparison.OrdinalIgnoreCase) ||
                             details.Contains("Could not list modules", StringComparison.OrdinalIgnoreCase) ||
+                            details.Contains("Operation not permitted", StringComparison.OrdinalIgnoreCase) ||
```

Comment:
When do the cases that you have added here happen?

Reply:
I ran into these additional cases while onboarding 1ES-hosted pools in the `release/1.1` branch. That was a couple months back so I don't remember details, but I hit these errors directly, debugged them, and determined that I needed to filter them here. I suspect they were just one or two of the many different things contributing to test instability at that time.

I don't know whether these cases are somehow specific to `release/1.1`, but left them here for consistency.